justfile

```diff
@@ -302,9 +302,7 @@ run_maker_args := if os() == "linux" {
 }
 
 maker args="":
-    # we always delete the old container first as otherwise we might get an error if the container still exists
-    docker rm -f maker || true
-    docker run {{run_maker_args}}
+    cargo run --bin dev-maker
```

Comment:
🤔 I guess we will have to add the dev makers pubkey to the `whitelisted_makers` in the `test-coordinator-settings.toml`

Reply:
For local dev setup it shouldn't be needed because whitelisting should be disabled.